service/routes.py

```diff
@@ -97,6 +97,21 @@ def read(promotion_id):
     app.logger.info("Returning promotion: %s", promotion.promotion_name)
     return jsonify(promotion.serialize()), status.HTTP_200_OK
 
+
+@app.route("/promotions/<int:promotion_id>", methods=["DELETE"])
+def delete(promotion_id):
+    """Deletes a Promotion with promotion_id with the fields included in the body of the request"""
+    app.logger.info(f"Got request to delete Promotion with id: {promotion_id}")
+    promotion = Promotion.find(promotion_id)
+    if not promotion:
+        abort_with_error(status.HTTP_404_NOT_FOUND,
+                         f"Promotion with id: {promotion_id} not found")
+    try:
+        promotion.delete()
+        return jsonify({}), status.HTTP_200_OK
```

Comment:
I think for successful delete we should return 204 instead of 200

Reply:
Done.